src/Assets/Bugsnag/Editor/BugsnagEditor.cs

```diff
@@ -6,12 +6,21 @@
 using BugsnagUnity;
 using UnityEditor.Callbacks;
 using System.Linq;
+using System;
 
 namespace BugsnagUnity.Editor
 {
     public class BugsnagEditor : EditorWindow
     {
 
+        private const string ANDROID_DEPS_XML = "<dependencies><androidPackages><repositories><repository>https://mvnrepository.com/artifact/org.jetbrains.kotlin/kotlin-stdlib</repository></repositories><androidPackage spec=\"org.jetbrains.kotlin:kotlin-stdlib:1.5.0\"></androidPackage></androidPackages></dependencies>";
```

Comment:
I believe Android is on v1.4.32 of `kotlin-stdlib` - see [Versions.kt](https://github.com/bugsnag/bugsnag-android/blob/next/buildSrc/src/main/kotlin/com/bugsnag/android/Versions.kt#L14) and the Rakefile for Unity uses v1.3.72.

Are we deliberately targeting a newer version?

Reply:
@lemnik got any insight on this? Should i be targeting 1.4.32?

Reply:
Yup. I think 1.4.32 is a better option.

Reply:
I updated the versions everywhere to use 1.4.32, and added comments regarding keeping them up to date